.gitignore

```diff
@@ -0,0 +1 @@
+*.sublime-*
```

Comment:
This is fairly popular repo's .gitignore:
https://github.com/twbs/bootstrap/blob/master/.gitignore